8510Final-AR.qmd

```diff
@@ -0,0 +1,311 @@
+---
+title: "Final Project"
+author: "Megha Khanna"
+date: "05/01/2024"
+---
+# Title of paper here
+
+For my Final project, I have applied codes to analyze a dataset of newspaper publications concerning Jack the Ripper and his notorious murders.
```

Comment:
Rather than "I have applied various codes", I think you want to say "I have used digital methodologies such as topic modeling, to analyze....."